### PR TITLE
generate new version of  all packages

### PR DESCRIPTION
**Pull request by mkholjuraev**
Generates new version of all packages.

### CODE REVIEW

File: packages/host-inventory/api.ts [hunk 1]
```diff
@@ -2885,7 +2922,7 @@ export interface YumRepo {
 export const AccountsStalenessApiAxiosParamCreator = function (configuration?: Configuration) {
     return {
         /**
-         * Create account staleness record. Required permissions: inventory:TODO:write
+         * Create account staleness record. Required permissions: staleness:staleness:write
```
Comment (bastilian) [inline packages/host-inventory/api.ts:2925]: hm. what happened here? It seems that only the docs changed, but I wonder where it gets these permissions from.

Comment (mkholjuraev) [reply packages/host-inventory/api.ts:2925]: hmm, I have no context on this. The generation works automatically. I will investigate why.

Comment (mkholjuraev) [reply packages/host-inventory/api.ts:2925]: BTW, this should be due to the change in the OpenAPI spec by the backend team

Comment (mkholjuraev) [reply packages/host-inventory/api.ts:2925]: Yes, it is due to an update in the API spec: https://developers.redhat.com/api-catalog/api/inventory

Comment (bastilian) [reply packages/host-inventory/api.ts:2925]: Oh, yeah! Thank you for investigating this! Should we maybe point the backend team at this. It doesn't seem correct, or?

Comment (mkholjuraev) [reply packages/host-inventory/api.ts:2925]: Yup, reported

**Review by apinkert — APPROVED** [review]
LGTM assuming you get the expected API spec changes in host-inventory!

**Pull request merged**